Controllers/BookmarksController.cs

```diff
@@ -0,0 +1,95 @@
+using bootcamp_api.Models;
+using bootcamp_api.Services;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace bootcamp_api.Controllers;
+
+[ApiController]
+[Produces("application/json")]
+[Consumes("application/json")]
+[SwaggerResponse(500, "An error occurred while processing the request", typeof(ProblemDetails))]
+[Route("[controller]")]
+public class BookmarksController : ControllerBase
+{
+    public BookmarksController()
+    {
+    }
+    
+    /// <summary>
+    /// Returns all bookmarks
+    /// </summary>
+    [HttpGet]
+    [SwaggerResponse(200, "The bookmark(s) are returned.", typeof(Bookmark[]))]
+    public ActionResult<List<Bookmark>> GetAll() =>
+        BookmarkService.GetAll();
+
+    
+    /// <summary>
+    /// Returns a bookmark with a given id
+    /// </summary>
+    [HttpGet("{id}")]
+    [SwaggerResponse(200, "The bookmark is returned.", typeof(Bookmark))]
+    [SwaggerResponse(404, "A bookmark with the given id does not exist.", typeof(ProblemDetails))]
+    public ActionResult<Bookmark> Get(int id)
+    {
+        var bookmark = BookmarkService.Get(id);
+
+        if (bookmark == null)
+            return NotFound();
+
+        return bookmark;
```

Comment:
You should use the helper `Ok(bookmark)`

Reply:
This issue was fixed in both the pets and bookmarks controllers.